artwork-data/managedPOIs.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
What's the point of this file? Can you mention it in the readme or in a wiki page perhaps?

Reply:
ok